### ISSUE TITLE
Page does not have a descriptive heading - Taxonomy term pages

### ISSUE THREAD

**Issue by cory-skeers** [uiowa/uiowa#9547]
Relates to #9541

Views-based pages for taxonomy terms use the term as the title. A lot of these pages have the potential to be flagged as "Page does not have a descriptive heading."

Example: https://clas.uiowa.edu/tags/rr-news

In some cases like that, it's the result of a more internal-focused tagging name. In others, it may be a descriptive name for the term, but not the page ("Science," or "Undergraduate").
